src/lib/api-client.ts

```diff
@@ -631,29 +644,69 @@ export class APIClient {
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async fetchMinors(subscriberId: string, namespace: string, name: string, major: number): Promise<Minor[]> {
+  async fetchMinors(subscriberId: string, namespace: string, name: string, major: string): Promise<Minor[]> {
     const response =  await this.get<MinorResults>(`/relaypro/api/v1/task_types/${namespace}/${name}/majors/${major}/minors?subscriber_id=${subscriberId}`)
     return response.body.results
   }
 
+  async fetchMajor(subscriberId: string, namespace: string, name: string, major: string): Promise<Major> {
+    let response
```

Comment:
I feel this can be simplified to:

```
const response = await this.get<Major>(`/relaypro/api/v1/task_types/${namespace}/${name}/majors/${major}?subscriber_id=${subscriberId}`)
```

Reply:
You're definitely right lol